bridge-token-factory/tests/token_transfer.rs

```diff
@@ -63,161 +84,218 @@ impl From<Proof> for AugmentedProof {
+    .to_string();
+    assert_eq!(alice_balance, "\"0\"");
+
+    let total_supply: String = std::str::from_utf8(
+        &rt.block_on(worker.view(
+            &token_account_id.parse().unwrap(),
+            "ft_total_supply",
+            json!({}).to_string().into_bytes(),
+        ))
+        .unwrap()
+        .result,
+    )
+    .unwrap()
+    .to_string();
+    assert_eq!(total_supply, "\"0\"");
 
     let mut proof = Proof::default();
     proof.log_entry_data = EthLockedEvent {
         locker_address: validate_eth_address(LOCKER_ADDRESS.to_string()),
         token: DAI_ADDRESS.to_string(),
         sender: SENDER_ADDRESS.to_string(),
         amount: 1_000,
-        recipient: ALICE.parse().unwrap(),
+        recipient: user.id().to_string().parse().unwrap(),
     }
     .to_log_entry_data();
 
-    call_borsh!(user, factory_id.deposit(proof)).assert_success();
-
-    let alice_balance: String =
-        call_json!(user, token_account_id.ft_balance_of({"account_id": ALICE.to_string()}))
-            .unwrap_json();
-    assert_eq!(alice_balance, "1000");
-
-    let total_supply: String = call_json!(user, token_account_id.ft_total_supply({})).unwrap_json();
-    assert_eq!(total_supply, "1000");
-
-    call_json!(alice, token_account_id.withdraw({
-                "amount" : "100",
-                "recipient" : SENDER_ADDRESS.to_string()}), deposit=1)
-    .assert_success();
-
-    let alice_balance: String =
-        call_json!(user, token_account_id.ft_balance_of({"account_id": ALICE.to_string()}))
-            .unwrap_json();
-    assert_eq!(alice_balance, "900");
-
-    let total_supply: String = call_json!(user, token_account_id.ft_total_supply({})).unwrap_json();
-    assert_eq!(total_supply, "900");
+    assert!(&rt
+        .block_on(
+            user.call(&worker, factory.id(), "deposit")
+                .deposit(parse_near!("50 N"))
+                .max_gas()
+                .args(proof.try_to_vec().unwrap())
+                .transact()
+        )
+        .unwrap()
+        .is_success());
+
+    let alice_balance: String = std::str::from_utf8(
+        &rt.block_on(
+            worker.view(
+                &token_account_id.parse().unwrap(),
+                "ft_balance_of",
+                json!({"account_id": user.id().to_string()})
+                    .to_string()
+                    .into_bytes(),
+            ),
+        )
+            .unwrap()
+            .result,
+    ).unwrap().to_string();
+    assert_eq!(alice_balance, "\"1000\"");
+
+    let total_supply: String = std::str::from_utf8(
+        &rt.block_on(worker.view(
+            &token_account_id.parse().unwrap(),
+            "ft_total_supply",
+            json!({}).to_string().into_bytes(),
+        )).unwrap().result,
+    ).unwrap().to_string();
+    assert_eq!(total_supply, "\"1000\"");
+
+    assert!(&rt.block_on(
+        user.call(&worker, &token_account_id.parse().unwrap(), "withdraw")
+            .max_gas()
+            .deposit(1)
```

Comment:
please use `ONE_YOCTO` instead

Reply:
fixed: https://github.com/aurora-is-near/rainbow-token-connector/pull/159/commits/ed254b0cde8629d0d51fe21949c2110bf19c5978